### PR TITLE
Add data explorer transforms

### PR DESCRIPTION
**Pull request by BenRussert**
This is more or less working and will be a great feature. The main issues I notice are style based, but I am hoping to get some others to test this out.

```python
import pandas as pd
pd.options.display.html.table_schema = True
pd.options.display.max_rows = None


iris_path =  "./iris.csv"

df1 = pd.read_csv(iris_path)

df1

```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,6 +72,7 @@
     "react-hot-loader": "^4.3.3",
     "react-rangeslider": "^2.1.0",
     "spawnteract": "^5.0.0",
+    "styled-jsx": "^2.2.7",
```
Comment (BenRussert) [inline package.json]: See https://github.com/nteract/nteract/issues/3113

**Comment by BenRussert** [comment]
hydrogen-data-explorer1
hydrogen-data-explorer2
hydrogen-data-explorer3

**Comment by kylebarron** [comment]
This looks amazing. With a dark theme some of the menus are hard to see. I'm not sure if that's Hydrogen's fault or the theme's fault:
image

**Comment by kylebarron** [comment]
Also, on Mac vertical two-finger scrolling isn't correctly intercepted by the DataFrame.

**Comment by BenRussert** [comment]
Thanks @kylebarron , if you think it's good enough for now you can merge this here. Then we could keep track of the various data explorer things to work on in a new issue.

**Comment by kylebarron** [comment]
That sounds fine. Especially since the user would have to type `pd.options.display.html.table_schema = True`, it seems unlikely to bother users at this stage.

**Comment by BenRussert** [comment]
I agree completely. Click that green button whenever you're ready @kylebarron !

**Comment by kylebarron** [comment]
I don't have write access to the repository...?

**Comment by BenRussert** [comment]
Woops, fixed that 😀

**Comment by kylebarron** [comment]
I now have write access to nteract/nteract, but still not to nteract/hydrogen.

**Comment by rgbkrk** [comment]
Our `dream-team` didn't have access to Hydrogen. 😅  That was my fault. Looks like all is set. Welcome and thanks @kylebarron!

**Pull request merged**